libsql-wal/src/name.rs

```diff
@@ -3,7 +3,7 @@ use std::fmt;
 use bytes::Bytes;
 
 #[derive(Clone, PartialEq, Eq, Hash)]
-pub struct NamespaceName(Bytes);
+pub struct NamespaceName(pub Bytes);
```

Comment:
just expose a `from<Bytes>` rather than make it pub?

Reply:
it's a placeholder type, it's going away asap

Reply:
I'm thinking I'll push the namespace type to the wal crate, and consume it in libsql-server. This will be a followup